<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreeSSM_MainWindow</class>
 <widget class="QMainWindow" name="FreeSSM_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>341</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>604</width>
    <height>341</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>604</width>
    <height>341</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
   </font>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exit_pushButton">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>286</y>
      <width>172</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>          E&amp;xit              </string>
    </property>
    <property name="icon">
     <iconset resource="../resources/FreeSSM.qrc">
      <normaloff>:/icons/oxygen/32x32/system-shutdown.png</normaloff>:/icons/oxygen/32x32/system-shutdown.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="controlunits_groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>565</width>
      <height>129</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Control Units:</string>
    </property>
    <widget class="QPushButton" name="engine_pushButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>24</y>
       <width>172</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>        &amp;Engine         </string>
     </property>
     <property name="icon">
      <iconset resource="../resources/FreeSSM.qrc">
       <normaloff>:/icons/freessm/64x64/Engine.png</normaloff>:/icons/freessm/64x64/Engine.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="transmission_pushButton">
     <property name="geometry">
      <rect>
       <x>197</x>
       <y>24</y>
       <width>172</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>   &amp;Transmission  </string>
     </property>
     <property name="icon">
      <iconset resource="../resources/FreeSSM.qrc">
       <normaloff>:/icons/freessm/64x64/Transmission.png</normaloff>:/icons/freessm/64x64/Transmission.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="absvdc_pushButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>74</y>
       <width>172</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>      ABS/VDC       </string>
     </property>
     <property name="icon">
      <iconset resource="../resources/FreeSSM.qrc">
       <normaloff>:/icons/freessm/64x64/ABS.png</normaloff>:/icons/freessm/64x64/ABS.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="cruisecontrol_pushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>197</x>
       <y>74</y>
       <width>172</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>  Cruise Control </string>
     </property>
     <property name="icon">
      <iconset resource="../resources/FreeSSM.qrc">
       <normaloff>:/icons/freessm/64x64/CruiseControl.png</normaloff>:/icons/freessm/64x64/CruiseControl.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="aircon_pushButton">
     <property name="geometry">
      <rect>
       <x>379</x>
       <y>74</y>
       <width>172</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Air Conditioning</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/FreeSSM.qrc">
       <normaloff>:/icons/freessm/64x64/AirCon.png</normaloff>:/icons/freessm/64x64/AirCon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="program_groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>194</y>
      <width>565</width>
      <height>79</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Program:</string>
    </property>
    <widget class="QPushButton" name="help_pushButton">
     <property name="geometry">
      <rect>
       <x>197</x>
       <y>24</y>
       <width>172</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>         &amp;Help             </string>
     </property>
     <property name="icon">
      <iconset resource="../resources/FreeSSM.qrc">
       <normaloff>:/icons/oxygen/32x32/help-browser.png</normaloff>:/icons/oxygen/32x32/help-browser.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="about_pushButton">
     <property name="geometry">
      <rect>
       <x>379</x>
       <y>24</y>
       <width>172</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>         &amp;About           </string>
     </property>
     <property name="icon">
      <iconset resource="../resources/FreeSSM.qrc">
       <normaloff>:/icons/oxygen/32x32/help-about.png</normaloff>:/icons/oxygen/32x32/help-about.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="preferences_pushButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>24</y>
       <width>172</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>    &amp;Preferences    </string>
     </property>
     <property name="icon">
      <iconset resource="../resources/FreeSSM.qrc">
       <normaloff>:/icons/oxygen/32x32/preferences-system.png</normaloff>:/icons/oxygen/32x32/preferences-system.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="background_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>604</width>
      <height>341</height>
     </rect>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>background_label</zorder>
   <zorder>exit_pushButton</zorder>
   <zorder>controlunits_groupBox</zorder>
   <zorder>program_groupBox</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>engine_pushButton</tabstop>
  <tabstop>transmission_pushButton</tabstop>
  <tabstop>preferences_pushButton</tabstop>
  <tabstop>help_pushButton</tabstop>
  <tabstop>about_pushButton</tabstop>
  <tabstop>exit_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/FreeSSM.qrc"/>
 </resources>
 <connections/>
</ui>
